tools/gitignore/gitignore.py

```diff
@@ -26,110 +26,112 @@
 
 
 def fnmatch_translate(pat):
-    # type: (str) -> Tuple[bool, Pattern[str]]
+    # type: (bytes) -> Tuple[bool, Pattern[bytes]]
```

Comment:
> but it's a bigger change

Could you elaborate a bit why it'd be a bigger change? It seems to me that the change would be about the same, since you've touched basically every string in `gitignore.py` anyway. Or are there more places that'd get affected if we use text strings?

> b) helped find an existing bug.

`append` instead of `+=`? https://github.com/web-platform-tests/wpt/pull/23733/files#diff-b47aa6e6b1fdd60fba3ddc80409e1d67R75

Reply:
Alright then.